### PR TITLE
Link to in-a-box

### PR DESCRIPTION
**Pull request by technetos**
This PR adds a section to the bottom of the Volunteer page about running your own rustbridge chapter and adds links in the Resources section to the various in-a-box resources.

### CODE REVIEW

File: data/volunteer.md [hunk 1]
```diff
@@ -40,3 +40,7 @@ space for events, or catering.
 
 Sponsors will get prominent display on event pages as well as the option to
 distribute swag at events.
+
+#### Interested in hosting your own RustBridge chapter?
+
+See the resources section for awesome tips, outlines and guides!
```
Comment (oe) [inline data/volunteer.md]: this could be a link, i feel

Comment (technetos) [reply data/volunteer.md]: Make 
#### Interested in hosting your own RustBridge chapter? 

link directly to the in-a-box repo?

Comment (oe) [reply data/volunteer.md]: @technetos good idea!

**Comment by technetos** [comment]
I have made the requested changes

File: data/volunteer.md [hunk 2]
```diff
@@ -41,6 +41,6 @@ space for events, or catering.
 Sponsors will get prominent display on event pages as well as the option to
 distribute swag at events.
 
-#### Interested in hosting your own RustBridge chapter?
+#### [Interested in hosting your own RustBridge chapter?][1]
```
Comment (oe) [inline data/volunteer.md]: maybe also rework `your own` into `a local`?

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**